src/js/themes/getStyle.js

```diff
@@ -212,6 +214,14 @@ const getDefaultThemeStyling = theme => {
             marginRight: constants.iconMarginRight,
             verticalAlign: 'top',
         },
+        'copy-icon-copied': {
+            color: colors.copyToClipboardCheck,
+            marginLeft: constants.clipboardCheckMarginLeft
+        },
+        'array-group-meta-data': {
+            display: 'inline-block',
+            padding: constants.arrayGroupMetaPadding
+        },
```

Comment:
i'd like `object-size` to return `cursor: pointer` to indicate that it's clickable